controllers/userController.js

```diff
@@ -0,0 +1,26 @@
+
+const userService = require('../services/userService');
+const { catchAsync } = require('../middleware/error');
+
+const signInKakao = catchAsync(async (req,res) => {
+    const { kakaoToken } = req.body;
+
+    if(!kakaoToken) throw new Error(401, 'need_kakaotoken');
```

Comment:
가독성을 위해서 띄어쓰기를 한 번해주세요! 
```suggestion
    if (!kakaoToken) throw new Error(401, 'need_kakaotoken');
```